Lib/turtle.py

```diff
@@ -2989,7 +2989,7 @@ def _getshapepoly(self, polygon, compound=False):
             t11, t12, t21, t22 = l, 0, 0, l
         elif self._resizemode == "noresize":
             return polygon
-        return tuple((t11*x + t12*y, t21*x + t22*y) for (x, y) in polygon)
+        return tuple([(t11*x + t12*y, t21*x + t22*y) for (x, y) in polygon])
```

Comment:
The list comprehension is not necessary and doesn't improve the code. This should stay as-is.